website/content/en/docs/deployment/_index.md

```diff
@@ -3,3 +3,12 @@ title = "Deployment"
 description = "Deploy Kubeflow on AWS"
 weight = 10
 +++
+Kubeflow on AWS is differentiated from the open-source KubeFlow project by a custom build manifest that provides integrations with various AWS services that are highly available and scalable. This reduces the operational overhead of maintaining the Kubeflow platform. 
+
+If you want to deploy Kubeflow with minimal changes, but optimized for [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks/) (Amazon EKS), then consider the [vanilla](/kubeflow-manifests/docs/deployment/vanilla/guide/) deployment option. The Kubeflow on AWS control plane is installed on top of Amazon EKS, which is a managed container service used to run and scale Kubernetes applications in the cloud.
```

Comment:
The Kubeflow on AWS control plane -> Kubeflow control place

Also want to convey, use this option if you you want to quickly try it out